game/server/eventstream/models.py

```diff
@@ -19,7 +19,6 @@
 
 class GameUpdateEventsJs(BaseModel):
     updated_flight_positions: dict[UUID, LeafletPoint]
-    new_combats: list[FrozenCombatJs]
```

Comment:
I can get behind removing the new/updated distinction in the slice since the front-end doesn't currently care, but let's keep it in the API. We do have plans to use that in the future (think "new combat has begin" type notifications).

Reply:
Wilco

Reply:
Small catch though, you can't go half-way with this because it'll cause validation errors. I think it's simpler to leave `new_combats` both in front-& back-end, especially if there are still plans to use it...